.github/labeler.json

```diff
@@ -1,53 +1,65 @@
 {
   "labels": {
-    "area: examples": ["examples/**"],
-    "area: documentation": ["docs/**", "errors/**"],
     "area: create-next-app": ["packages/create-next-app/**"],
-    "type: next": [
-      "packages/next/**",
-      "packages/react-dev-overlay/**",
-      "packages/react-refresh-utils/**",
-      "packages/next-codemod/**",
-      "packages/eslint-plugin-next/**",
-      "packages/eslint-config-next/**",
-      "packages/next-env/**",
-      "packages/next-swc/**"
+    "area: documentation": ["docs/**", "errors/**"],
+    "area: examples": ["examples/**"],
+    "area: next/image": [
+      "**/*image*",
+      "**/*image*/**",
+      "packages/next/src/client/use-intersection.tsx",
+      "packages/next/src/server/lib/squoosh/",
+      "packages/next/src/server/serve-static.ts"
     ],
+    "area: Font Optimization": ["**/*font*"],
```

Comment:
We could auto-label more features to be able to filter on this while looking for PRs to review, but some features are spread across multiple files that often change, so it's not always easy to keep track of them.

For this reason, I only added `next/image` and `next/font` for now.